react-app/src/api/distributionAPI.ts

```diff
@@ -67,16 +72,57 @@ export const useDistributionAPI = (): IDistributionAPI => {
     );
 
     return {
-      denom: coinMinimalDenom,
+      denom: CoinDenom,
       amount: convertMinimalTokenToToken(rewardAmount),
     };
-  }, [coinMinimalDenom, wallet, query]);
+  }, [wallet, query]);
+
+  const getTotalCommission = useCallback(async () => {
+    if (wallet.status !== ConnectionStatus.Connected) {
+      throw new Error("Wallet not connected");
+    }
+    if (!wallet.account.address.includes("valoper")) {
```

Comment:
seems more suitable to decode the bech32 and compare the prefix a config value

Reply:
I took another look at this, turns out we need to translate `like1...` address to `likevaloper1...` first, then use the `likevaloper1...` address to query commissions. In this case, checking for bech32 prefix is nolonger needed.